test/functional/helpers/createFixture/clientScripts.js

```diff
@@ -74,7 +74,7 @@ const getProdNpmLibraryCode = () => {
   return readCache.sync(prodNpmLibraryPath, "utf8");
 };
 
-const injectInlineScript = ClientFunction(code => {
+export const injectInlineScript = ClientFunction(code => {
```

Comment:
should we extract this explicitly in a test util function? 
cc: @jonsnyder

Reply:
I don't think extracting it to a separate test util is warranted, since it's already used in other `clientScripts.js` functions, like `injectAlloyDuringTestForInt()`

Reply:
Good question Nina. I probably would have added a parameter to "createFixture" for extra scripts to include. I think this is fine though. There are already two exports for clientScripts.js.